src/components/Sentence/SentenceCard.tsx

```diff
@@ -18,8 +18,12 @@ const StyledComment = styled(Comment)`
 `;
 
 const SentenceCard = ({ sentence }) => {
-    const { content, claim, personality } = sentence;
-    const contentModel = sentence?.claim[0].contentModel;
+    const {
+        content,
+        claim: [claim],
```

Comment:
issue: I don't see the point of converting to array just to get the first element later. You should check if it's an array or not and try to access the contentModel based on that